docs/Inputs.md

```diff
@@ -530,12 +530,12 @@ const theme = {
         ...defaultTheme.components,
         MuiTextField: {
             defaultProps: {
-                variant: 'outlined' as const,
+                variant: 'outlined',
```

Comment:
Good catch! but as the TypeScript validation causes problem, I suggest you add a tip just after this snipper for TS users.